proposals/PROPOSALS.md

```diff
@@ -1,141 +1,127 @@
-2. **Implementation guidance:**  the myriad of subject matter experts and stakeholders affected by a proposal can comment in a centralized place to form a consensus and guide the implementation strategy for a particular task.
-3. **Resource requirements:** ideas are often discussed in the abstract, without the consideration for the cost of work required for their implementation. This process allows for discussion that reifies the resource requirements for proposals, and allows the community to more accurately weigh the cost and benefits of an implementation.
+Collectively, the documents that serve as the basis for these discussions are called _Haskell Foundation Technical Proposals_ (HFTP).
+Additionally, at its option, the TWG will host projects and discussions that don't directly fit into the above categories, but that seem to be useful for the community as a whole.
 
-It's important to note that seeing a proposal through to its conclusion is an involved task. On the one hand, it takes time to convince people that your suggestions are a worthwhile change for hundreds of thousands of developers to accept. Particularly given the sheer volume of developers that could be affected by a proposal, its acceptance is conservative and carefully thought through. Typically, this includes many rounds of discussion with HF leaders, Haskell library maintainers, and the broader community, several iterations on the design of the proposal, and some effort at prototyping the proposed change. Often, it takes weeks to months of discussion, re-design, and prototyping for a proposal to be accepted. It is therefore important to note that seeing a proposal through to its conclusion can be time-consuming and not all proposals may end up accepted, although they may teach us all something!
+The TWG will address the following concerns:
+ * Soliciting projects from the community, where relevant
+ * Ensuring that relevant stakeholders have been contacted and that their input and needs have been considered
+ * Guiding discussions in a kind and productive direction
+ * Providing a final recommendation as to whether a project should continue
+If the TWG does not reach a consensus, then the working group's decision will be made by majority vote.
 
-If you’re motivated enough to go through this involved but rewarding process, go on with writing and keep on reading.
+Projects undertaken by the Haskell Foundation need not go through this process, and approval of a project by this working group does not guarantee that the Haskell Foundation will actually commit resources. Committee members are tasked with recommending whether a project is likely to be of benefit to the community, not for prioritizing the Foundation's limited resources. However, we expect that, given sufficient resources, recommended projects will be supported by the executive team.
```

Comment:
I don't think that we need to expand "likely of benefit" here because we discuss possible reasons for rejection or acceptance elsewhere and also leave this to the discretion of the committee. Clearly an unactionable proposal is not of benefit, for example. We could, however, to account for cost, say "likely of net benefit".

I strongly disagree with the second point regarding limiting criteria. For one, on the acceptance side it makes promises that the TWG can't keep, because we're not the ones committing the resources or writing the fundraiser emails! For another, on the limiting side having a backlog of six proposals or whatever might work better for fundraising. That said, the alternative is not "organic" and "whatever happens happens." The alternative is that the director gives us instructions, based on the concerns of the HF at the moment, and we act in accord with those instructions. So it is still managed and controlled rather than accidental -- it just is that we do not have the charter control it, rather we have the leadership of the HF control it as they best see fit.

Reply:
I agree with @gbaz here.

The overarching point I'd like to make here is that this document is not a formal specification, nor is it an algorithm. It's also not a contract between parties with an adversarial relationship. It's a set set of guidelines intended to help actual people working together in good faith to apply their knowledge, wisdom, and emotional intelligence to help the Haskell community, and especially the HF, make wise decisions about limited resources.

 On the first point, I don't think that unactionable proposals are likely to be of benefit, so that's enough reason to ask for a revision (as there could very well be a useful, actionable proposal with a bit of work and feedback).

On the second point, I think we can better manage it by having me maintain a public backlog of projects that I select from the proposals and put on the HF web page, rather than by having the contents of a Git repo serve as if it were public communication. What do you think?

Reply:
Fair enough -- I withdraw my suggestions.